understanding/language-of-parts.html

```diff
@@ -48,7 +48,7 @@ <h2>意図</h2>
                   <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注</div><p>2 級点字はグレード 2 とも呼ばれる。これは、英語の点字において、使用頻度の高い単語又は単語の一部を短縮した特別な点字を用いるものである。詳細については、<a href="https://ja.wikipedia.org/wiki/%E8%8B%B1%E8%AA%9E%E3%81%AE%E7%82%B9%E5%AD%97">英語の点字</a>及び<a href="http://www.dinf.ne.jp/doc/japanese/resource/blind/eigohyoki_rewrite.html">日本における英語点字の表記について</a>を参照のこと。</li>
                
                
-               <li>複数の言語をサポートしている合成音声は、適切な発音で適切なアクセントのテキストを話すことができるようになる。言語の変化が示されていない場合、合成音声は、動作するデフォルトの言語として単語を読み上げるのに最善を尽くそうとする。したがって、自動車を表すフランス語の単語 &quot;voiture&quot; は、デフォルトの言語として英語を用いている合成音声では &quot;voyture&quot; と発音されるだろう。</li>
+						<li>複数の言語をサポートしている合成音声は、適切な発音で適切なアクセントのテキストを話すことができるようになる。言語の変化が示されていない場合、合成音声は、動作するデフォルトの言語として単語を読み上げるのに最善を尽くそうとする。したがって、自動車を表すフランス語の単語 &quot;<span lang="fr">voiture</span>&quot; は、デフォルトの言語として英語を用いている合成音声では &quot;voyture&quot; と発音されるだろう。</li>
```

Comment:
どうというわけではないですが、文字参照を元に戻しておきます

```suggestion
						<li>複数の言語をサポートしている合成音声は、適切な発音で適切なアクセントのテキストを話すことができるようになる。言語の変化が示されていない場合、合成音声は、動作するデフォルトの言語として単語を読み上げるのに最善を尽くそうとする。したがって、自動車を表すフランス語の単語 "<span lang="fr">voiture</span>" は、デフォルトの言語として英語を用いている合成音声では "voyture" と発音されるだろう。</li>
```